controllers/webhook/runcompletion_feed_decoupled_test.go

```diff
@@ -0,0 +1,121 @@
+//go:build decoupled
+
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"io"
+	"net/http"
+	"net/http/httptest"
+)
+
+var counter = 0
```

Comment:
perhaps a better name could be `mockUpstreamServiceCallCounter` ?

Reply:
See [d626eca](https://github.com/sky-uk/kfp-operator/pull/368/commits/d626ecacae21d54eb78ccd9c1dcb1075cbef2215)